src/main/java/knight2103/Ui.java

```diff
@@ -0,0 +1,67 @@
+package knight2103;
+
+import knight2103.tasks.Task;
+import knight2103.tasks.TaskList;
+
+import java.util.Scanner;
```

Comment:
import order is wrong. Refer to the standards for the order

Reply:
Oh did the coding standards mandate any order? I thought so long as it is consistent, it's okay already.